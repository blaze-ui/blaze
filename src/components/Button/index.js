/* eslint-disable react/destructuring-assignment */
/* eslint-disable react/button-has-type */
/* eslint-disable react/prefer-stateless-function */
import React from "react";
import Proptypes from "prop-types";
import "./Button.css";

class Button extends React.Component {
  render() {
    return (
      <div>
        <button className={this.props.color}>{this.props.children}</button>
      </div>
    );
  }
}

Button.propType = {
  color: Proptypes.string
};
export default Button;
