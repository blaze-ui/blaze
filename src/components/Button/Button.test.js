import React from 'react';
import renderer from 'react-test-renderer';
import Button from './index';

test('Button default test case', () => {
    const component = renderer.create(
        <Button color="primaryButton"> Hello </Button>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

