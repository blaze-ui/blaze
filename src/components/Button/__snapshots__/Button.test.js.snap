// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button default test case 1`] = `
<div>
  <button
    className="primaryButton"
  >
     Hello 
  </button>
</div>
`;
